cmd/notifier-config.go

```diff
@@ -235,10 +235,7 @@ func (n *notifier) Validate() error {
 	if err := n.MySQL.Validate(); err != nil {
 		return err
 	}
-	if err := n.MQTT.Validate(); err != nil {
-		return err
-	}
-	return nil
+	return n.MQTT.Validate()
```

Comment:
I suggest rewriting this function with closures to remove some repetition and to restore the uniformity of checking that was present for all notification targets. This would work:

```go
func (n *notifier) Validate() error {
	if n == nil {
		return nil
	}

	var err error
	chkErr := func(f func() error) {
		if err == nil {
			err = f()
		}
	}

	chkErr(n.AMQP.Validate)
	chkErr(n.NATS.Validate)
	chkErr(n.ElasticSearch.Validate)
	chkErr(n.Redis.Validate)
	chkErr(n.PostgreSQL.Validate)
	chkErr(n.Kafka.Validate)
	chkErr(n.Webhook.Validate)
	chkErr(n.MySQL.Validate)
	chkErr(n.MQTT.Validate)
	return err
}

```